data/vocab/scholar.ttl

```diff
@@ -469,6 +469,35 @@ scholar:mostCitedWork a rdf:Property ;
     rdfs:range lite:Work ;
     rdfs:domain lite:Person .
 
+scholar:totalWorks a rdf:Property ;
+    rdfs:label "total works"@en ;
+    rdfs:comment "The total number of works authored or co-authored by the person, as reported by external sources."@en ;
+    rdfs:domain lite:Person ;
+    rdfs:range xsd:integer .
+
+scholar:totalUniqueCoAuthors a rdf:Property ;
+    rdfs:label "total unique co-authors"@en ;
+    rdfs:comment "The total number of distinct authors with whom the person has collaborated across all their works. This count includes only unique individuals, regardless of the number of joint works."@en ;    rdfs:domain lite:Person ;
+    rdfs:range xsd:integer .
+
+scholar:totalWorksWithMultipleContributors a rdf:Property ;
+    rdfs:label "total works with multiple contributors"@en ;
+    rdfs:comment "The total number of works with multiple contributors authored or co-authored by the person."@en ;
+    rdfs:domain lite:Person ;
+    rdfs:range xsd:integer .
+
+scholar:Contribution a rdfs:Class ;
+    rdfs:label "Contribution"@en ;
+    rdfs:comment "Represents a contribution by a person to a work with additional metadata such as order."@en ;
+    rdfs:subClassOf rdf:Statement .
+
+scholar:contributor a rdf:Property ;
+    rdfs:label "contributor"@en ;
+    rdfs:comment "Entity (or entities) who contribute to the origin resource."@en ;
+    rdfs:domain scholar:Contribution ;
+    rdfs:range lite:Agent ;
+    owl:equivalentProperty schema:contributor, dc:contributor .
+
```

Comment:
this is in lite, don't need it defined here
```suggestion
```

Reply:
lite contributor does not have relation to Contribution. not to affect lot of classes, I thought maybe creating scholar contributor with contribution was required